File: letter/ru/Ю.py
from common import *

expr = Eq(
    y
    * (x + S(1) / 2)
    * (16 * x ** 2 - 8 * x + y ** 2)
    * sqrt(
        ((x + S(1) / 2 + eps) * (16 * x ** 2 - 8 * x + y ** 2 + eps)) /
        ((1 - Abs(y)) * (S(1) / 2 + eps - x))
    )
    ,
    0
)
size = 1

File: letter/ru/Ю.js
import { Letter } from '../../Letter.js';

const expr = 'y * (x + 1/2) * (16*x^2 - 8*x + y^2) * sqrt(((x + 1/2 + 1/10) * (16*x^2 - 8*x + y^2 + 1/10)) / ((1 - abs(y)) * (1/2 + 1/10 - x)))';

export const letter = new Letter(expr, 1);
